java/format/pom.xml

```diff
@@ -23,20 +23,16 @@
 <name>Arrow Format</name>
 
   <properties>
-    <fbs.version>1.2.0-3f79e055</fbs.version>
     <flatc.download.skip>false</flatc.download.skip>
     <flatc.executable>${project.build.directory}/flatc-${os.detected.classifier}-${fbs.version}.exe</flatc.executable>
     <flatc.generated.files>${project.build.directory}/generated-sources/flatc</flatc.generated.files>
-    <maven-compiler-plugin.version>3.3</maven-compiler-plugin.version>
-    <maven-dependency-plugin.version>2.10</maven-dependency-plugin.version>
```

Comment:
re the dependency plugin - I defined the dependency plugin globals in the ```pluginManagement``` in the parent pom, this way it gets propagate to all instances of this plugin in child poms.

re the compiler - see my separate answer above. if it is desirable to stick it to particular version I would recommend putting it into the ```pluginManagement``` section of the parent similarly to what I did with the dependency plugin.